docs/kyma/03-05-resource-quotas.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Resource quotas
+type: Details
+---
+
+[Resource quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/) are a convenient way to manage the consumption of resources in a Kyma cluster. You can easily set resource quotas for every Namespace you create through the Console UI.
+
+When you click **Create Namespace**, you can define:
+  - Total Memory Quotas, which limit the overall memory consumption by the Namespace by creating a ResourceQuota object.
+  - Limits per Container, which limit the memory consumption for individual Containers in the Namespace by creating a [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/) object.
```

Comment:
```suggestion
  - Limits per container, which limit the memory consumption for individual containers in the Namespace by creating [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/) objects.
```

Reply:
As far as I remember, we do not capitalize containers.

Reply:
It's weird to capitalize Pods but not containers, but okay ;p